hasura/metadata/databases/ibcjuno/tables/public_token_ibc_denom.yaml

```
@@ -0,0 +1,18 @@
+table:
+  name: token_ibc_denom
+  schema: public
+object_relationships:
+- name: token_unit
```

I think we can change this to be simply `unit` for easier usage 
```suggestion
- name: unit
```